setup.py

```diff
@@ -41,6 +41,7 @@
     "flask-paginate>=2022.1.8",
     "Flask-WTF>=1.0.1",
     "Flask>=2.2.2,<2.3",
+    "WTForms-JSON>=0.3.5",
```

Comment:
The file `bukuserver/requirements.txt` must be kept in sync with `setup.py`.

Reply:
i didn't have the impression it was kept in sync because of the comment on top:
`# use setup.py for latest required package `

Reply:
[There's a test for it.](https://github.com/jarun/buku/blob/v4.8/tests/test_setup.py#L17-L22)